docs/guidance/implementation.md

```diff
@@ -0,0 +1,51 @@
+
+## Standalone infrastructure transparency portal
+
+![Standalone infrastructure transparency portal](../_static/images/implementation_scenario_1.png)
+
+In this model, procuring entities enter project and contract level data directly into an infrastructure transparency portal and the portal publishes project data and summary contracting process data in OC4IDS format. CoST Honduras uses this model in [SISOCS](https://sisocs.org/), as does CoST West Lombok in [INTRAS](http://intras.fllajlombokbaratkab.or.id/).
+
+The main benefit of this approach is that it is relatively simple because it does not involve integrating data from different systems. The downside of this approach is that it can increase the data entry burden on procuring entities; for example, if they also need to enter the data about contracting processes into a separate procurement system.
+
+## Integrated infrastructure transparency portal and procurement system
+
+![Integrated infrastructure transparency portal and procurement system](../_static/images/implementation_scenario_4.png)
+
+In this model, procuring entities enter project data directly into an infrastructure transparency portal and the portal imports data about contracting processes from a separate procurement system. The infrastructure transparency portal publishes project data and summary contracting process data in OC4IDS format.
+
+For each project, the infrastructure transparency portal needs to join up the project data entered by the procuring entity with contracting data imported from the procurement system. If the contracting data includes [project identifiers](https://standard.open-contracting.org/infrastructure/latest/en/guidance/identifiers/#local-project-identifiers-in-contracting-data), this process can be automated. Otherwise, procuring entities need to manually associate contracts with projects.
+
+CoST Ukraine uses this model in its [infrastructure transparency portal](https://portal.costukraine.org/), which imports data from Prozorro, the national procurement system. Project identifiers are not captured in [Prozorro](https://prozorro.gov.ua/en) so procuring entities manually match contracts to projects.
+
+The main benefit of this approach is reducing the data entry burden on procuring entities, because they need only enter contracting data in one system. When the data from the procurement system includes project identifiers, the burden is further reduced since the transparency portal can match contracts to projects without manual intervention. The downsides of this approach are increased complexity of the transparency portal and the potential need for development work on the procurement system to enable access to its data.
```

Comment:
```suggestion
The main benefit of this approach is reducing the data entry burden on procuring entities, who need only enter contracting data in one system. When the data from the procurement system includes project identifiers, the burden is further reduced since the transparency portal can match contracts to projects without manual intervention. The downsides of this approach are increased complexity of the transparency portal and the potential need for development work on the procurement system to enable access to its data.
```